Makefile

```diff
@@ -82,7 +82,14 @@ run: my.env  ## | Run processor, webapp, fakesentry, symbolsserver, and required
 
 .PHONY: runservices
 runservices: my.env  ## | Run service containers (Postgres, SQS, etc)
-	${DC} up -d statsd postgresql memcached localstack elasticsearch symbolsserver
+	${DC} up -d \
```

Comment:
We also want to add `--remove-orphans` which should have been here earlier. That will clean up local dev environments when switching between branches which have different services.